nbs/01_read.ipynb

```diff
@@ -35,10 +35,10 @@
     "from fastcore.xdg import *\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #49.                              {'title':yaml_str(title)}, {'description':yaml_str(desc)} if desc else {})
</pre><p>Does <code>yaml_str</code> solve a specific problem here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/803/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
Yes, this is dealing with quotes, since we are extracting the YAML with regex we need to also make sure its valid YAML